input/resources/au-core-server.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CapabilityStatement xmlns="http://hl7.org/fhir">
```

Comment:
Remove support for http://hl7.org/fhir/uv/sdc/ImplementationGuide/hl7.fhir.uv.sdc

Reply:
Done